third_party/brotli/c/enc/memory.h

```diff
@@ -85,7 +85,7 @@ R: requested size
     while (_new_size < (R)) _new_size *= 2;                        \
     new_array = BROTLI_ALLOC((M), T, _new_size);                   \
     if (!BROTLI_IS_OOM(M) && !BROTLI_IS_NULL(new_array) && C != 0) \
-      memcpy(new_array, A, C * sizeof(T));                         \
+      memcpy(new_array, A, C * sizeof(T));                  \
```

Comment:
The backslash at EOL needs 7 more spaces indentation to match upstream commit e61745a6.